.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and publish @prisma/prisma-fmt-wasm
+
+concurrency: build-prisma-fmt-wasm
+
+on:
+  workflow_dispatch:
+    inputs:
+      enginesWrapperVersion:
+        required: true
+      enginesHash:
+        required: true
+      npmDistTag:
+        required: true
+        default: 'latest'
```

Comment:
Who triggers this? Engines-Wrapper repo? Somewhere in this PR (I can not find `enginesWrapperVersion` here anywhere, so not sure)?

Reply:
The input shape is exactly the same as that of the existing publish workflow in the prisma-fmt-wasm repo. It's meant to be triggered by engines-wrapper in exactly the same way.

Reply:
Found this below:
> It is triggered from the https://github.com/prisma/engines-wrapper publish action.

So the loop is now:
Engine `main` commit => Engine-wrapper CI (does unrelated stuff) and triggers => Engines CI (this workflow) => published to Npm?

Reply:
yep. One big difference is that the publish pipeline does not commit anymore.

Reply:
The only thing it might want to commit now is the version in package.json. It's easy to correlate with the engines hash though, because they are part of the same input in a job. We could also trivially include the engines git commit hash as part of the `version()` output of prisma-fmt-wasm now.

Reply:
Yep, no commit is good.